src/main/java/AddCommand.java

```diff
@@ -0,0 +1,60 @@
+public class AddCommand extends Command {
+
+    private String input;
+    private int counter;
+    private int type;
+
+    AddCommand(Tasks[] tasks, String input, int counter, int type) {
+        super(tasks);
+        this.input = input;
+        this.counter = counter;
+        this.type = type;
+    }
+
+    public Tasks[] addTask() {
+        if (type == 0) {
```

Comment:
Using numbers 0, 1, and 2 to represent task type of event, deadline, and todo would be a bit confusing. Please consider changing it.